bedrock/settings/static_media.py

```diff
@@ -1133,7 +1132,6 @@
         'source_filenames': (
             'js/base/mozilla-fxa-iframe.js',
             'js/base/uitour-lib.js',
-            'js/firefox/sync-animation.js',
```

Comment:
Looks like this might be the end of this lib? Can we remove it?

Reply:
Yup, will remove!